docs/components/console/manage-clusters/cluster-capacity.md

```diff
@@ -0,0 +1,104 @@
+
+Cluster capacity provides a high-level overview of how well a cluster is coping with its current workload.
+
+- Use this information to check and monitor if a cluster is appropriately sized for its workload.
+- Cluster capacity can also be used as an indicator of cluster health. For example, a cluster running at maximum capacity can be an indicator of poor cluster responsiveness.
+
+A general guideline to follow when using cluster capacity as a metric is:
+
+- **High capacity percentage**: The higher the capacity percentage value, the more likely it is that things will slow down, time out, requests will fail, and so on.
+- **Low capacity percentage**: The lower the capacity percentage value, the more you could increase the cluster workload as the cluster is probably underused.
+
+For example, if a cluster capacity is continually running at 95%, this means it is probably overloaded and may not be performing well, but if the cluster capacity is only 5% then it can probably accept more workload and may be underused.
+
+:::info
+To understand how cluster capacity is calculated, see [how cluster capacity is calculated](#capacity-calculation).
+:::
+
+## View cluster capacity
+
+The cluster capacity percentage is shown in the **Capacity** column on the Console **Clusters** tab.
+
+<img src={CapacityPercentImg} alt="Example cluster capacity percentage" />
+
+Select a cluster to view detailed cluster capacity information on the cluster **Overview** tab.
+
+<img src={CapacityImg} alt="Example cluster capacity" style={{width: '550px'}}/>
+
+- The current cluster capacity is shown as a percentage bar at the top of the section.
+- The chart shows detailed capacity data for the last 24 hours, 7 days, or 30 days.
+  - Select the time period you want to view data for.
+  - Hover over individual nodes in the chart to view data for a specific time or day.
+- Click the **Refresh** icon to refresh and update the cluster capacity data.
+
+## Manage cluster capacity
+
+Cluster capacity will fluctuate based on incoming user requests, and internal processing load.
+
+- User requests are those sent directly by an external client.
+- Internal load refers to all other processing that is **not** directly triggered by a client. For example, timer events, a job being made available after back off, and so on.
+
+This means that cluster capacity could fluctuate throughout the day. For example:
+
+- Capacity might be higher during business hours, but lower overnight when the cluster is unused.
+- Certain processes can cause a "fan-out" effect, where even though creating a process instance is only a single user request, it could require computation on a very large multi-instance collection, resulting in a high capacity spike.
+- You could have a spike in capacity if a large number of timers are triggered at the same time.
+
+### High capacity
+
+A high capacity does not necessarily mean that a cluster needs to be resized, but it could indicate that your cluster is overloaded.
```

Comment:
:wrench: Perhaps we should distinguish between capacity and utilization? "A high capacity" to me sounds like a really big cluster that can handle a lot of throughput, whereas "A high utilization" makes it clear that there's little _free_ capacity and the cluster is close to being overloaded.